rpc/gnmi/gnmi-specification.md

```diff
@@ -688,13 +680,12 @@ In the case that any operation within the `SetRequest` message fails, then (as p
 
 For the operation that the target is unable to process, the `message` field MUST be set to a specific error code indicating the reason for failure based on the following mappings to canonical gRPC error codes:
 
-
-
 *   When the client has specified metadata requiring authentication (see [Section 3.1](#31-session-security-authentication-and-rpc-authorization)), and the authentication fails -  `Unauthenticated (16)`.
 *   When the client does not have permission to modify the path specified by the operation - `PermissionDenied (7)`.
 *   When the operation specifies a path that cannot be parsed by the target - `InvalidArgument (3)`. In this case, the `message` field of the `Error` message specified SHOULD specify human-readable text indicating that the path could not be parsed.
 *   When the operation is an update or replace operation that corresponds to a path that is not valid - `NotFound (5)`. In this case the `message` field of the `Error` message specified SHOULD specify human-readable text indicating the path that was invalid.
-*   When the operation is an update or replace operation that includes an invalid value within the `Update` message specified - `InvalidArgument (3)`. This error SHOULD be used in cases where the payload specifies scalar values that do not correspond to the correct schema type, and in the case that multiple values are specified using a particular encoding (e.g., JSON) which cannot be decoded by the target.
+*   When the operation is an update or replace operation that includes an invalid value within the `Update` message specified - `InvalidArgument (3)`. This error SHOULD be used in cases where the payload specifies scalar values that do not correspond to the correct schema type,
+* When the client specifies a payload utilising an encoding that is not supported by the target (e.g., JSON) - `Unimplemented (12)` SHOULD be used to indicate that the encoding is not supported.
```

Comment:
this could be a bit confusing -- wouldn't Unimplemented be more likely to be interpreted as the path not being supported?  InvalidArgument might be a better choice (i.e., the encoding is invalid, esp. if Capability has been used, but even if not)